saas_public/README.rst

```diff
@@ -0,0 +1,40 @@
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.png
+   :target: https://www.gnu.org/licenses/agpl
+   :alt: License: AGPL-3
+
+===============
+ SaaS Public
+===============
+
+This module can be useful when creating public builds for ready-made templates by link.
```

Comment:
Create builds for non-authenticated users.

* User opens a link and gets a build